src/Corrade/Utility/DebugStl.h

```diff
@@ -130,4 +149,14 @@ CORRADE_UTILITY_EXPORT Debug& operator<<(Debug& debug, Implementation::DebugOstr
 
 }}
 
+template<typename T>
```

Comment:
One last thing: can you 

- add a short documentation block for this new function,
- mention this in the class documentation so people get to know it exists -- there's a "Printing STL types" section, so maybe extending that a bit ("Printing STL types, interaction with iostreams"?)
- and add a test in `Test/DebugTest.cpp`? There's `ostreamFallback` and `ostreamFallbackPriority`, so something similar (`ostreamDelegate`?) and also similarly checking the priority works for the inverse case, picking the ostream operator over the Debug one -- btw., don't forget to list the two new methods in the `addTests()` so these get properly called

Thank you! :+1:

Reply:
yes! wow `ostreamDelegate` is way better than any name I considered